projects/components/src/data-exporter/data-exporter.component.ts

```diff
@@ -131,8 +131,8 @@ export class DataExporterComponent implements OnInit {
 
     private _open = false;
 
-    _shouldPurifyData = false;
-    _data: any[][];
+    shouldSanitizeData = false;
```

Comment:
This seems to really mean  `dataHasPotentialInjection` since we should only sanitize the data if the user chooses Yes.